sdk/resourcemanager/Azure.ResourceManager/samples/Generated/Samples/Sample_TenantCollection.cs

```diff
@@ -42,25 +41,5 @@ public async Task GetAll_GetAllTenants()
 
             Console.WriteLine("Succeeded");
         }
-
-        [Test]
-        [Ignore("Only validating compilation of examples")]
-        public async Task CheckResourceName_CheckValidityForAResourceName()
-        {
-            // Generated from example definition: specification/resources/resource-manager/Microsoft.Resources/stable/2022-12-01/examples/CheckResourceName.json
-            // this example is just showing the usage of "CheckResourceName" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `checkResourceName`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/resources/resource-manager/Microsoft.Resources/stable/2015-11-01/subscriptions.json#L199